boards/FSM/Src/App/App_AcceleratorPedals.c

```diff
@@ -28,25 +30,24 @@ struct AcceleratorPedals
  */
 static float App_GetPedalPercentage(
     uint32_t encoder_fully_pressed_value,
+    uint32_t encoder_reset_value,
     uint32_t encoder_counter_value,
     void (*reset_encoder_counter)(void));
 
 static float App_GetPedalPercentage(
     uint32_t encoder_fully_pressed_value,
+    uint32_t encoder_reset_value,
     uint32_t encoder_counter_value,
```

Comment:
can we use a pound define in this file to specify the `encoder_fully_pressed` and `encoder_counter` values?

Reply:
fully pressed + reset value are currently defined in FSM\Inc\App\configs\App_AcceleratorPedalThresholds.h